src/compiler/scala/tools/nsc/backend/jvm/LabelNode1.java

```diff
@@ -13,7 +13,6 @@
 package scala.tools.nsc.backend.jvm;
 
 import scala.tools.asm.Label;
-import scala.tools.asm.tree.ClassNode;
```

Comment:
What is this java? and have they no tooling to detect unused imports?

Reply:
Not sure if they have no tooling or not but I've found more unused imports since.